core/templates/dev/head/domain/topic/TopicUpdateService.js

```diff
@@ -195,13 +195,21 @@ oppia.factory('TopicUpdateService', [
        *    created in the current draft of the topic. (i.e, the subtopic to
        *    delete hasn't been saved in the datastore yet.)
        */
-      deleteSubtopic: function(topic, subtopicId, isNewlyCreated) {
+      deleteSubtopic: function(topic, subtopicId) {
```

Comment:
Always remember to update the docstrings, please.

@apb7 @kevinlee12 we should detect function signature / docstring mismatches.

Reply:
Done.